fava/templates/_layout.html

```diff
@@ -101,6 +95,16 @@ <h1 class="site-name">{{ api.title or 'fava' }}{{ ' ▾' if config['APIS']|lengt
     </article>
     {% include "overlays/_keyboard_shortcuts.html" %}
     {% include "overlays/_documents_upload.html" %}
+    {% if config['APIS']|length > 1 %}
```

Comment:
Because of the way z-indexing works, it became impossible for that dropdown to display above the aside menu because it was declared in the header tag before the aside tag.

The other option, I suppose, would have been to move the entire aside tag above the header, but that solution seemed weirder than moving this one down.